src/View/Helper/FormHelper.php

```diff
@@ -359,12 +366,16 @@ public function create($model = null, array $options = [])
             'encoding' => strtolower(Configure::read('App.encoding')),
             'templates' => null,
             'idPrefix' => null,
+            'valuesSources' => $this->getValuesSources(),
```

Comment:
We could potentially reset state in end(). We do that for secured fields for example.